src/shogun/base/SGObject.cpp

```diff
@@ -741,3 +763,24 @@ CSGObject* CSGObject::clone()
 	SG_DEBUG("leaving %s::clone(): Clone successful\n", get_name());
 	return copy;
 }
+
+void CSGObject::set_with_base_tag(const BaseTag& _tag, const Any& any)
+{
+	self->set(_tag, any);
+}
+
+Any CSGObject::get_with_base_tag(const BaseTag& _tag) const
+{
+	Any any = self->get(_tag);
+	if(any.empty())
+	{
+		SG_ERROR("There is no parameter called \"%s\" in %s",
+			_tag.name().c_str(), get_name());
+	}
+	return any;
+}
+
+bool CSGObject::has_with_base_tag(const BaseTag& _tag) const
+{
+	return self->has(_tag);
+}
```

Comment:
Missing line in several files...
I just saw them yesterday and I can't help making them right..